monai/transforms/croppad/array.py

```diff
@@ -158,9 +158,9 @@ def __call__(self, img: np.ndarray, mode: Optional[Union[NumpyPadMode, str]] = N
         spatial_border = tuple(max(0, b) for b in spatial_border)
 
         if len(spatial_border) == 1:
-            data_pad_width = [(spatial_border[0], spatial_border[0]) for _ in range(len(spatial_shape))]
+            data_pad_width = [(spatial_border[0], spatial_border[0]) for _ in spatial_shape]
         elif len(spatial_border) == len(spatial_shape):
-            data_pad_width = [(spatial_border[i], spatial_border[i]) for i in range(len(spatial_shape))]
+            data_pad_width = [(sp, sp) for sp, _ in zip(spatial_border, spatial_shape)]
```

Comment:
`data_pad_width = [(sp,) * 2 for sp in spatial_border[:len(spatial_shape)]]` 

Is that clearer or actually worse?